From: Hanjun Guo <guohanjun@huawei.com>
Date: Thu, 24 Sep 2020 10:57:03 +0800
Subject: ACPI: proc: Remove dead ACPICA debug code
Patch-mainline: v5.10-rc1
Git-commit: d93b767e8cb319b3062d2e6ef4328bee97da57f5
References: jsc#SLE-16407

Remove the not used ACPICA debug code _COMPONENT and ACPI_MODULE_NAME()
which were not used even when proc.c was introduced.

Signed-off-by: Hanjun Guo <guohanjun@huawei.com>
[ rjw: Subject edit ]
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/proc.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/acpi/proc.c
+++ b/drivers/acpi/proc.c
@@ -10,15 +10,11 @@
 #include "sleep.h"
 #include "internal.h"
 
-#define _COMPONENT		ACPI_SYSTEM_COMPONENT
-
 /*
  * this file provides support for:
  * /proc/acpi/wakeup
  */
 
-ACPI_MODULE_NAME("sleep")
-
 static int
 acpi_system_wakeup_device_seq_show(struct seq_file *seq, void *offset)
 {
